basic/s3_log_destination/s3_log_destination.tf

```diff
@@ -21,7 +21,7 @@ resource "aws_kinesis_firehose_delivery_stream" "sym_logs" {
   }
 
   tags = {
-    # This SymEnv tag is required and MUST match the SymEnv tag in the 
+    # This SymEnv tag is required and MUST match the SymEnv tag in the
     # aws_iam_policy.secrets_manager_access in your `secrets.tf` file
```

Comment:
I think this comment should be referencing the `kinesis_firehose_connector` module?